core-services/prow/02_config/openshift/cluster-api-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cluster-api-operator
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/cluster-api-operator
+  review_acts_as_lgtm: true
```

Comment:
should this be false?

Reply:
That was set by default, let's remove it.